wkcuber/cubing.py

```diff
@@ -157,7 +157,7 @@ def cubing_job(args):
                 for z, file_name in zip(z_batch, source_file_batch):
                     # Image shape will be (x, y, channel_count, z=1)
                     image = read_image_file(
-                        file_name, target_wkw_info.header.voxel_type
+                        file_name, target_wkw_info.header.voxel_type, z
```

Comment:
Is my understanding correct that, in case of non-multi-paged tiffs, the `z` is simply ignored because the `z` is also encoded in the filename?

Reply:
Correct